### PR TITLE
Commands added

### PR DESCRIPTION
**Pull request by TedSjoblom**
First examples of commands

### CODE REVIEW

File: messages/commands/playback.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            "description": "Reverse is number under zero. Zero is paused. Above zero is moving forward and 1 is seen as normal speed"
+        },
+        "start_from": {
+            "type": "number",
+            "title": "Start from",
+            "description": "Start frame where position from where the playback will start or continue from"
+        }
+      
+    }
+}
```
Comment (freol35241) [inline messages/commands/playback.json]: 👍

File: messages/commands/rudder_steering.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            "title": "Command id",
+            "description": "Platform unique command id"
+        },
+        "angle": {
+            "type": "number",
+            "title": "Rudder angle set",
+            "description": "Rudder angle in degrees, positive when trailing edge points to port"
+        }
+    }
+}
```
Comment (freol35241) [inline messages/commands/rudder_steering.json]: Just a control question:  Are these actual commands? I.e. they will be sent to control the rudder on some vessel/simulated vessel? Or are these just observations of set points?

File: messages/commands/rudder_steering.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Rudder Steering",
+    "description": "Command given for rudder steering unit",
+    "required": [
+        "observation_id",
```
Comment (freol35241) [inline messages/commands/rudder_steering.json]:
```suggestion
        "command_id",
```

File: messages/commands/rudder_steering.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        "angle"
+    ],
+    "properties": {
+        "command_id": {
```
Comment (freol35241) [inline messages/commands/rudder_steering.json]: `command_id` makes sense, so we should then also add the `actuators` description @luisheres was talking about in the review of #7.

File: messages/commands/water-jet_steering.json [hunk 5]
```diff
@@ -0,0 +1,27 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "type": "object",
+    "title": "Water-jet steering",
+    "description": "Command given for water-jet steering unit",
+    "required": [
+        "command_id",
+        "bucket_angle"
+    ],
```
Comment (freol35241) [inline messages/commands/water-jet_steering.json]:
```suggestion
    "required": [
        "command_id",
        "angle",
        "bucket_angle"
    ],
```

File: messages/commands/water-jet_steering.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    "properties": {
+        "command_id": {
+            "type": "string",
+            "title": "Command id",
+            "description": "Platform unique command id"
+        },
+        "angle": {
+            "type": "number",
+            "title": "Water-jet angle set",
+            "description": "Water-jet angle in degrees, positive when pushing water to starboard"
```
Comment (freol35241) [inline messages/commands/water-jet_steering.json]:
```suggestion
            "description": "Water-jet angle in degrees, positive when pushing water to port"
```

To conform to the same standard as the rudder.

File: messages/observations/ground_velocity.json [hunk 7]
```diff
@@ -19,6 +19,16 @@
...
+        "sog_transverse": {
+            "type": "number",
+            "title": "Speed over ground transverse [knots]",
+            "description": "Reported speed over ground transverse."
+        },
+        "sog_longships": {
+            "type": "number",
+            "title": "Speed over ground longships [knots]",
+            "description": "Reported speed over ground longships."
+        },
```
Comment (freol35241) [inline messages/observations/ground_velocity.json:31]: I still think this is redundant data, but perhaps, we could just introduce a new observation message `ground_velocity_transverse.json` to cater for the sensor which actually output these kind of redundant, human-friendly observations? Something like:
```json
{
    "$schema": "https://json-schema.org/draft/2020-12/schema",
    "type": "object",
    "title": "Ground velocity transverse",
    "description": "Reading from a sensor providing information about the transverse ground velocity",
    "required": [
        "observation_id",
        "sog",
    ],
    "properties": {
        "observation_id": {
            "type": "string",
            "title": "Observation id",
            "desciption": "Platform unique observation id"
        },
        "sog": {
            "type": "number",
            "title": "Speed over ground [knots]",
            "description": "Reported speed over ground."
        }
    }
}
```

**Review by freol35241 — COMMENTED** [review]
Nice work @TedSjoblom 

Some smaller comments and suggested fixes below.

File: messages/commands/rudder_steering.json (hunk 2 above)
Comment (TedSjoblom) [reply messages/commands/rudder_steering.json]: In my head😅, it is just the command from the user or script and the receiver should confirm the new set value by pushing an observation state ( hopefully this should be some form of close loop communication)

File: messages/commands/water-jet_steering.json (hunk 6 above)
Comment (TedSjoblom) [reply messages/commands/water-jet_steering.json]: Are port turns positive? Really? Not following the compass? This has to be some naval architect thing 😂

**Pull request merged**